### PR TITLE
Enable cargo clippy unused

### PR DESCRIPTION
**Pull request by Mikkel-Boje**

### CODE REVIEW

File: server/src/test_utils/websocket.rs [hunk 1]
```diff
@@ -211,16 +211,14 @@ impl WSStream for MockSocket {
...
 #[cfg(test)]
 pub(crate) mod test {
-    use std::time::Duration;
-
     use super::MockSocket;
-    use super::WSStream;
     use axum::extract::ws::Message;
     use futures_util::SinkExt;
-    use futures_util::{stream::SplitStream, StreamExt};
+    use futures_util::StreamExt;
```
Comment (UnluckyBird) [inline server/src/test_utils/websocket.rs:217]: `use futures_util::{SinkExt, StreamExt};`

**Review by UnluckyBird — APPROVED** [review]

File: client/src/websockets.rs [hunk 2]
```diff
@@ -38,10 +38,10 @@ use tokio_tungstenite::{
...
 const SECOND: u32 = 1000;
 const MINUTE: u32 = SECOND * 60;
-const HOUR: u32 = MINUTE * 60;
-const DAY: u32 = HOUR * 24;
-const WEEK: u32 = DAY * 7;
-const MONTH: u32 = DAY * 30;
+// const HOUR: u32 = MINUTE * 60;
+// const DAY: u32 = HOUR * 24;
+// const WEEK: u32 = DAY * 7;
+// const MONTH: u32 = DAY * 30;
```
Comment (alexsteffensen) [inline client/src/websockets.rs:44]:
```suggestion
```

File: client/src/websockets.rs [hunk 3]
```diff
@@ -54,7 +54,7 @@ const STALE_THRESHOLD_MS: u32 = 5 * MINUTE;
 // close the socket.
 const KEEPALIVE_TIMEOUT_MS: u32 = 30 * SECOND;
 
-const MAX_MESSAGE_SIZE: u32 = 512 * 1024;
+// const MAX_MESSAGE_SIZE: u32 = 512 * 1024;
```
Comment (alexsteffensen) [inline client/src/websockets.rs:57]:
```suggestion
```

File: server/src/test_utils/websocket.rs (hunk 1 above)
Comment (Mikkel-Boje) [reply server/src/test_utils/websocket.rs]: Fixed

File: client/src/websockets.rs (hunk 2 above)
Comment (Mikkel-Boje) [reply client/src/websockets.rs]: Removed

File: client/src/websockets.rs (hunk 3 above)
Comment (Mikkel-Boje) [reply client/src/websockets.rs]: Removed

**Review by andreasalstrup — APPROVED** [review]

**Pull request merged**